tests.yaml

```diff
@@ -44,3 +44,31 @@ tests:
       exp_annotations:
         message: "CoreDNS has 99th percentile latency of 8.109262626262627 seconds for server dns://:53 zone . ."
         runbook_url: "https://github.com/povilasv/coredns-mixin/tree/master/runbook.md#alert-name-corednslatencyhigh"
+
+- interval: 1m
+  input_series:
+    - series: 'coredns_forward_healthcheck_failure_count_total{instance="1.2.3.4:9153",job="kube-dns",kubernetes_name="coredns-kube-metrics",kubernetes_namespace="kube-system",pod="coredns-65b6759cb4-qgdxp",server="dns://:53",zone=".",to="5.6.7.8"}'
+      values: '100'
```

Comment:
add more values as right now you have only a value at t=0, t=1m will have no value.